UPGRADE_GUIDE.md

```diff
@@ -5,6 +5,107 @@ To see discussion around these API changes, please refer to the
 [changelog](/CHANGELOG.md) and visit the commits and issues they
 reference.
 
+0.13.x -> 1.0.0
+----------------
+
+- React `0.14.x` is now required
```

Comment:
This issue was the deal-breaker that required us to upgrade to 0.14 - since we wanted to use the AsyncProps feature. So technically speaking, to use all features for React Router 1.0 - you need React 0.14.

https://github.com/rackt/react-router/issues/1511